labs/emerald-performance-testharness.md

```diff
@@ -0,0 +1,52 @@
+# Lab Name
```

Comment:
Should the file be named `labs/emerald-performance-test-harness.md`?  In other words, do you want a dash between `test` and `harness`. Remember, the name of the markdown file will be the name of the lab.